lumicks/pylake/tests/test_import_time.py

```diff
@@ -0,0 +1,21 @@
+from textwrap import dedent
+import numpy as np
+import subprocess
+import os
+
+
+def test_disabling_capturing(capsys):
+    repeats = 3
+
+    code = dedent("""\
+        import time
+        tic = time.time()
+        import lumicks.pylake
+        print(time.time() - tic)
+        """)
+
+    times = [float(subprocess.check_output(['python', '-c', code])) for i in range(repeats)]
+
+    with capsys.disabled():
+        print(os.environ['TEST_IMPORT_TIME'])
```

Comment:
You can use `pytest.mark` for this. E.g. see `pytest.mark.slow`: https://pytest.org/en/stable/example/simple.html#control-skipping-of-tests-according-to-command-line-option

Reply:
👍